src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/13 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings2.sql

```diff
@@ -102,3 +105,22 @@ SELECT
 				ELSE CONVERT(VARCHAR(10), '07/31/' +  Cast(CalendarYear as varchar) , 101) END
 				From  Reference.CollectionPeriods Where [Open] = 1)
 		 And   ndtt.TransactionType In ( 13,14,15)
+GO
+
+IF EXISTS (
+		SELECT 1
+		FROM [sys].[indexes] i
+		JOIN sys.objects t ON i.object_id = t.object_id
+		WHERE t.name = 'ApprenticeshipEarnings2'
+		AND i.[name] = 'IX_ApprenticeshipEarnings2_UKPRN'
+		)
+BEGIN
+	DROP INDEX [IX_ApprenticeshipEarnings2_UKPRN]
+		ON [Staging].[ApprenticeshipEarnings2]
+END
+
+CREATE CLUSTERED INDEX [IX_ApprenticeshipEarnings2_UKPRN] ON [Staging].[ApprenticeshipEarnings2]
+(
+	[Ukprn] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, SORT_IN_TEMPDB = OFF, DROP_EXISTING = OFF, ONLINE = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+GO
```

Comment:
there is a step in the payments process that indexes these three tables and rather than vw_apprenticeshipearning looking at apprenticeshipearning1, 2, 3, it looks at a new table that contains all the data from the three tables.

Therefore, remove index.